docs/_guide/use.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: guide
+title: Use a component
+slug: use
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+This page describes how to [use a LitElement component in your application](#use). It also describes how to make sure your deployed code is browser-ready by [building it for production](#build) and [loading the Web Components polyfills](#polyfills).
+
+## Use a LitElement component {#use}
+
+This is a general guide to using third-party LitElement components. Refer to a component's README or other documentation for specific details.
+
+To use a LitElement component in your code:
+
+1.  Install the component from npm.
+
+    ```
+    npm install --save some-package-name
+    ```
+
+2.  Import the component.
+
+    In a JavaScript module: 
+
+    ```js
+    import 'some-package-name';
+    ```
+
+    In an HTML page:
+
+    ```html
+    <script type="module">
+    import './path-to/some-package-name/some-component.js';
+    </script>
+    ```
+
+    Or:
+
+    ```html
+    <script type="module" src="./path-to/some-package-name/some-component.js"></script>
+    ```
+
+3.  Add the component to the page via markup:
+
+    ```html
+    <some-component></some-component>
+    ```
+
+## Build for production {#build}
+
+LitElement is published on npm using JavaScript Modules. This means it can take advantage of the standard native JavaScript module loader available in all current major browsers. 
+
+You will need a light transform to resolve LitElement's npm dependencies. This can be done with a bundler such as WebPack or Rollup.
+
+The following example configuration for [Rollup](https://rollupjs.org/guide/en) resolves modules and dependencies, and bundles the output.
+
+**rollup.config.js**
+
+```js
+import resolve from 'rollup-plugin-node-resolve';
+
+export default {
+  // If using any exports from a symlinked project, uncomment the following:
+  // preserveSymlinks: true,
+	input: ['src/index.js'],
+	output: {
+		file: 'build/index.js',
+    format: 'es',
+		sourcemap: true
+	},
+	plugins: [
+    resolve()
+  ]
+};
+```
+
+See a [sample build configuration for LitElement with Babel and Rollup](https://github.com/PolymerLabs/lit-element-build-rollup/blob/master/src/index.html).
+
+## Load the WebComponents polyfills {#polyfills}
+
```

Comment:
Maybe add an intro:

Since elements build with LitElement use the Web Components set of standards, which are not available on every browser yet, you will need to load the Web Components polyfills to support older browsers and Edge.